### PR TITLE
adcxx1c: adapt to new i2c API

### PR DESCRIPTION
**Pull request by vincent-d**
### Contribution description

Adapt adcxx1c driver to new i2c API.

### Issues/PRs references

#6577

### CODE REVIEW

File: drivers/adcxx1c/adcxx1c.c [hunk 1]
```diff
@@ -79,7 +71,7 @@ int adcxx1c_read_raw(const adcxx1c_t *dev, int16_t *raw)
     int status;
 
     i2c_acquire(I2C);
-    status = i2c_read_regs(I2C, ADDR, ADCXX1C_CONV_RES_ADDR, buf, 2);
+    status = i2c_read_regs(I2C, ADDR, ADCXX1C_CONV_RES_ADDR, buf, 2, 0);
     i2c_release(I2C);
     if (status < 2) {
```
Comment (smlng) [inline drivers/adcxx1c/adcxx1c.c]: the function does not return the read bytes anymore, but `0` on success

**Comment by dylad** [comment]
@vincent-d do you think you can addressed the comments soon ?

**Comment by vincent-d** [comment]
Sorry for the delay, my last 2 weeks have been busy.

Comment addressed.

**Comment by smlng** [comment]
please squash

**Comment by vincent-d** [comment]
Squashed

**Comment by dylad** [comment]
Thanks @vincent-d !
Let's merge this one.

**Pull request merged**